app/policies/person_policy.rb

```diff
@@ -0,0 +1,2 @@
+class PersonPolicy < BasePolicy
+end
```

Comment:
strange... Role is not Policy. Better EntityPolicy<BasePolicy   DictionaryEntityPolicy <DictionaryPolicy ect

Reply:
should I remove the inheritance from permission?

Reply:
yep. Use ApplicationPolicy as ancestor